sdks/java/core/src/main/java/org/apache/beam/sdk/io/Read.java

```diff
@@ -290,12 +289,17 @@ public void splitRestriction(
       long estimatedSize = restriction.getEstimatedSizeBytes(pipelineOptions);
       // Split into pieces as close to the default desired bundle size but if that would cause too
       // few splits then prefer to split up to the default desired number of splits.
-      long splitBundleSize =
-          Math.min(
-              DEFAULT_DESIRED_BUNDLE_SIZE_BYTES,
-              Math.max(1L, estimatedSize / DEFAULT_DESIRED_NUM_SPLITS));
```

Comment:
Should we delete the DEFAULT_DESIRED_NUM_SPLITS constant as well?

Reply:
It is used in the unbounded source wrapper, but I have little context on that.